controllers/rollingupgrade_controller.go

```diff
@@ -1049,13 +1049,15 @@ func (r *RollingUpgradeReconciler) DrainTerminate(
 			return err
 		}
 	}
+	ruObj.Status.LastNodeDrainTime = metav1.Time{Time: time.Now()}
```

Comment:
Please move this inside the if condition. In v1, we don't drain when a node isn't found.

```
// Drain and wait for draining node.
	if nodeName != "" {
		if err := r.DrainNode(ruObj, nodeName, targetInstanceID, ruObj.Spec.Strategy.DrainTimeout, nodeSteps, inProcessingNodes, mutex); err != nil {
			return err
		}
                ruObj.Status.LastNodeDrainTime = metav1.Time{Time: time.Now()}
	}
```

Reply:
Fixed